### PR TITLE
feature/117/create combat data

### PR DESCRIPTION
**Pull request by MGertz**

### CODE REVIEW

File: Common/src/main/java/dk/sdu/sesem4/common/data/combat/Weapon.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package dk.sdu.sesem4.common.data.combat;
+
+import dk.sdu.sesem4.common.data.entity.Entity;
+
+public class Weapon {
```
Comment (The0mikkel) [inline Common/src/main/java/dk/sdu/sesem4/common/data/combat/Weapon.java:5]: Missing JavaDoc

File: Common/src/main/java/dk/sdu/sesem4/common/data/combat/WeaponType.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package dk.sdu.sesem4.common.data.combat;
+
+public enum WeaponType {
```
Comment (The0mikkel) [inline Common/src/main/java/dk/sdu/sesem4/common/data/combat/WeaponType.java:3]: Missing JavaDoc

**Review by The0mikkel — CHANGES_REQUESTED** [review]

File: Common/src/main/java/dk/sdu/sesem4/common/data/combat/WeaponType.java (hunk 2 above)
Comment (MGertz) [reply Common/src/main/java/dk/sdu/sesem4/common/data/combat/WeaponType.java:6]: Added

**Review by JfriisJ — APPROVED** [review]
LGTM

**Pull request merged**